src/backend/gl/src/command.rs

```diff
@@ -108,6 +108,7 @@ pub enum Command {
     SetPatchSize(i32),
     BindProgram(<GlContext as glow::Context>::Program),
     BindBlendSlot(ColorSlot, pso::ColorBlendDesc),
+    BindAllBlendSlots(pso::ColorBlendDesc),
```

Comment:
Is this the right name, its not really binding...

Reply:
good point. Let's rename to `SetXxx`